apps/docs/.vitepress/config.ts

```diff
@@ -213,6 +213,10 @@ export const sidebar = [
             link: "/resources/integrations/multi-cms",
           },
           { text: "Strapi", link: "/resources/integrations/strapi/" },
+          {
+            text: "Custom Products extension",
+            link: "/resources/integrations/custom-products",
```

Comment:
I would put it below:
https://frontends.shopware.com/getting-started/features/

Because this is not really an integration.

Reply:
I've moved it to the mentioned chapter 👍🏼

Reply:
Nice then remove the old link from the sidebar also.